schedule/sles4sap/installation/agama_install_sles4sap.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: agama_install_sles4sap
+description: >
+  Agama installation tests for sles4sap
+
+  Can be used to generate a qcow2 image, used to test SAP components like
+  HANA, NetWeaver, WMP, etc.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/first_boot
+  - console/system_prepare
+  - '{{test_sles4sap}}'
+  - '{{generate_image}}'
+conditional_schedule:
+  test_sles4sap:
+    TEST_SLES4SAP:
+      1:
+        - sles4sap/patterns
+        - sles4sap/sapconf
```

Comment:
We can remove this line. There will be no sapconf in 16

Reply:
Fixed